### DISCUSSION TITLE
Reverse fake tls tunnel ایران

### DISCUSSION THREAD

**Discussion by MohammadAhmadiDev** [Gozargah/Marzban#537] [Show and tell]
سلام بنده با روش reverse fake tls tunnel یک سرور ایران رو به یکی از نود هام وصل کردم. بدون دخالت در مرزبان. الان ایپی سرور ایرانمو توی هر کانفیگی که با مرزبان ساختم بزارم وصل میشه. دقیقا چرا این اتفاق میوفته؟

**Comment by y8nes** [comment]
سلام بنده تست کردم, درست میگید اما اینباند های ریلیتی نود های دیگه همان آیپی نودی که تونل زده شده رو برمیگردونه. یعنی آیپی ایران رو اگر وصل کرده باشیم به نود آلمان و اونو توی نود آمریکا بزاریم به همون نود آلمان وصل میشه
